### PR TITLE
Switch to vanilla create-react-app now that it supports TypeScript

### PR DESCRIPTION
**Pull request by kfarnung**
Not in a hurry to land this, just curious what the change would look like.

### CODE REVIEW

**Comment by kfarnung** [comment]
Seems to work ok, we can decide whether it's worth landing later.  I'll need to test the service worker a bit.

File: src/Cipher/Caesar/CaesarList.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import React, { Component } from 'react';
```
Comment (beckbria) [inline src/Cipher/Caesar/CaesarList.tsx]: Why is React left out of the brackets?

File: package.json [hunk 2]
```diff
@@ -13,27 +13,33 @@
...
     "@types/react-bootstrap": "^0.32.14",
     "@types/react-dom": "^16.0.9",
     "@types/react-loadable": "^5.4.1",
     "@types/react-router-bootstrap": "^0.24.5",
     "@types/react-router-dom": "^4.3.1",
     "genversion": "^2.1.0",
     "typescript": "^3.1.6"
-  }
+  },
+  "browserslist": [
```
[the remark this answers is not in the document]
Comment (kfarnung) [reply package.json]: No idea, was generated by react.

File: src/Cipher/Caesar/CaesarList.tsx (hunk 1 above)
Comment (kfarnung) [reply src/Cipher/Caesar/CaesarList.tsx]: React is the name for the whole import, everything in the brackets is a specific export from the module.

**Pull request merged**